server/modules/shell_files/cpu_temp.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+if [ `which sensors` ]; then
```

Comment:
@Q-Max When `sensors` is not installed, the graph continues to show. Probably because there is no `else` block here as well as no empty response handler for the angular directive.

All widgets must show a message in the widget element if no data is present or the feature is not supported by the server.